MobileCenter/MSLogDBStorageTests.m

```diff
@@ -0,0 +1,587 @@
+#import <OCHamcrestIOS/OCHamcrestIOS.h>
```

Comment:
All the test classes are importing MSTestFrameworks.h in the target branch. You need to follow the rule to avoid warnings from macOS and tvOS tests.